src/AccordionItem/AccordionItem.js

```diff
@@ -48,7 +48,8 @@ class AccordionItem extends Component<AccordionItemProps, *> {
             ...rest
         } = this.props;
 
-        const currentItem = accordionStore.state.items.filter(
+        // Deliberately not using 'find' because IE compat.
```

Comment:
Ah ignore this comment then https://github.com/springload/react-accessible-accordion/pull/128/files#r225007869